src/test/java/org/broadinstitute/hellbender/tools/IndexFeatureFileIntegrationTest.java

```diff
@@ -265,6 +265,61 @@ public void testBedIndex() {
         Assert.assertEquals(index.getSequenceNames(), Arrays.asList("1", "2", "4"));
     }
 
+    // TODO: this is not enabled because canDecode returns false for bed.gz extension
+    // TODO: should this be changed at the htsjdk level to return true also for a block-compressed extension?
+    @Test(enabled = false)
+    public void testBedGZIndex() {
+        final File ORIG_FILE = getTestFile("test_bed_for_index.bed.gz");
+        final File outName = BaseTest.createTempFile("test_bed_for_index.bed.", TabixUtils.STANDARD_INDEX_EXTENSION);
```

Comment:
Can just write `createTempFile()` instead of `BaseTest.createTempFile()`, since it's inherited.